pkg/network/network.go

```diff
@@ -116,6 +117,13 @@ func Render(conf *opv1alpha1.NetworkAddonsConfigSpec, manifestDir string, opensh
 	}
 	objs = append(objs, o...)
 
+	// render Ovs
+	o, err = renderOvs(conf, manifestDir, clusterInfo)
+	if err != nil {
+		return nil, err
```

Comment:
fmt.Errorf to track the place where the error is generated.

Reply:
want me to change all the other components?

Reply:
Well then we just follow what it is, we can fix it at different PR